### PR TITLE
optimize: fix asf.yaml parse error

### LINKED ISSUE #6193: parse asf.yaml error
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

image

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

Minimal yet complete reproducer code (or URL to code):

### Ⅵ. Environment:

- JDK version(e.g. `java -version`):
- Seata client/server version:  
- Database version:
- OS(e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

fix asf.yaml parse error

### Ⅱ. Does this pull request fix one issue?

fix #6193

### CODE REVIEW

File: changes/en-us/2.x.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6178](https://github.com/apache/incubator-seata/pull/6178)] update the header of Apache License
 - [[#6186](https://github.com/apache/incubator-seata/pull/6186)] update README.md(update mailing list and repository urls)
 - [[#6184](https://github.com/apache/incubator-seata/pull/6184)] update NOTICE file
+- [[#6193](https://github.com/apache/incubator-seata/pull/6193)] fix asf.yaml parse error
```
Comment (lightClouds917) [inline changes/en-us/2.x.md]: the pr is 6194

File: changes/zh-cn/2.x.md [hunk 2]
```diff
@@ -34,6 +34,7 @@
 - [[#6178](https://github.com/apache/incubator-seata/pull/6178)] 更新Apache License头信息
 - [[#6186](https://github.com/apache/incubator-seata/pull/6186)] 更新README.md（更新邮件列表和一些生态访问链接）
 - [[#6184](https://github.com/apache/incubator-seata/pull/6184)] 更新NOTICE文件
+- [[#6193](https://github.com/apache/incubator-seata/pull/6193)] 修复 asf.yaml 解析错误问题
```
Comment (lightClouds917) [inline changes/zh-cn/2.x.md]: the pr is 6194

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

File: changes/en-us/2.x.md (hunk 1 above)
Comment (funky-eyes) [inline changes/en-us/2.x.md]:
```suggestion
- [[#6194](https://github.com/apache/incubator-seata/pull/6194)] fix asf.yaml parse error
```

File: changes/zh-cn/2.x.md (hunk 2 above)
Comment (funky-eyes) [inline changes/zh-cn/2.x.md]:
```suggestion
- [[#6194](https://github.com/apache/incubator-seata/pull/6194)] 修复 asf.yaml 解析错误问题
```

**Review by xingfudeshi — APPROVED** [review]
LGTM

**Pull request merged**